packages/manager/modules/telecom-dashboard/src/translations/Messages_en_AU.json

```
@@ -0,0 +1,40 @@
+{
+  "telecom_dashboard_title": "Summary of my services ",
```

Extra space but iso with XML.

```suggestion
  "telecom_dashboard_title": "Summary of my services",
```

---

Should be managed in another PR ;)

Here, we only want to switch from XML to JSON :)